content/articles/setting-up-a-private-dns-server-with-raspberry-pi/index.md

```diff
@@ -0,0 +1,132 @@
+For computers to speak with one another over the web, all participants within the network must have an IP address: Through these addresses, computers recognize precisely which servers/devices they are communicating with. However, it can be difficult to remember IP addresses and hence the need for DNS servers. Domain Name Systems (DNS) changes domain names into IP addresses. A DNS is like a phone book of web addresses that makes access to pages faster. 
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [How DNS servers work](#how-dns-servers-work)
+- [Functions of a DNS](#functions-of-a-dns)
+- [Setting up a DNS Server on Raspberry PI](#setting-up-a-dns-server-on-raspberry-pi)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+
+1.	Raspberry Pi (2-4) with Raspbian OS
+2.	Access to the internet
+3.	Access to the Raspberry's terminal
+
+### How DNS servers work
+
+Devices question one or more DNS servers to get access to the correct domain name – IP address translation, a process that can be time-consuming. Therefore, it would be beneficial to shorten the time spent on this process by having a private and dedicated DNS server. A Raspberry Pi, which is small but can be used for various purposes, would be beneficial in such a scenario.

```

Comment:
Please discuss how a DNS server works in depth. As it is it's very shallow. Use examples if necessary. Also, add an external link to Raspberry Pi's website